src/components/oscal-utils/OSCALProfileResolver.js

```diff
@@ -27,17 +27,19 @@ import getUriFromBackMatterByHref from "./OSCALBackMatterUtils";
  * @see {@link https://pages.nist.gov/OSCAL/documentation/schema/profile-layer/profile/xml-schema/#global_import}
  * @see {@link https://pages.nist.gov/OSCAL/documentation/schema/profile-layer/profile/xml-schema/#global_back-matter_h2}
  */
-export default function OSCALResolveProfileOrCatalogUrlControls(
+export default async function OSCALResolveProfileOrCatalogUrlControls(
```

Comment:
Is there a reason we're introducing asyncs and awaits for this?

Reply:
There was some issue when I was first doing the task that `async`/`await` seemed to resolve. It isn't needed I just forgot to remove them from the code.